lib/aytests/libvirt_vm.rb

```diff
@@ -0,0 +1,61 @@
+require "rexml/document"
+require "rexml/xpath"
+require "cheetah"
+
+module AYTests
+  class LibvirtVM
+    # Implements communication with Libvirt virtual machines.
```

Comment:
Same than below, I was expecting this comment one line before. But probably it's fine where it is.